src/pages/index.js

```diff
@@ -1,3 +1,10 @@
+import {
+  IconCopy,
+  IconDownload,
+  IconShare,
+  IconBrandTwitter
+} from "@tabler/icons";
```

Comment:
Are there a lot of icons here?

This could cause the bundle to include every icon. 

See https://nextjs.org/blog/next-13-1#import-resolution-for-smaller-bundles

Reply:
Will take a look if I can make the change

Reply:
@styfle I can't find another way to import these icons. Would you like to use a different icon library instead?

Reply:
I used the `react-icons` library which seems more performant. (Using Tabler Icons under the hood)
This is the build command output

```
Route (pages)                              Size     First Load JS
┌ ○ /                                      14.3 kB        93.4 kB
├   /_app                                  0 B            79.1 kB
├ ○ /404                                   181 B          79.3 kB
├ λ /api/v1/[username]                     0 B            79.1 kB
└ λ /api/v1/tweetMedia                     0 B            79.1 kB
+ First Load JS shared by all              80.9 kB
  ├ chunks/framework-114634acb84f8baa.js   45.4 kB
  ├ chunks/main-ee0cf4b7f81d7c24.js        27.1 kB
  ├ chunks/pages/_app-3adada8895e66558.js  5.82 kB
  ├ chunks/webpack-ee7e63bc15b31913.js     815 B
  └ css/671175cd27e07421.css               1.81 kB
```